auto_submit/lib/revert/revert_info_collection.dart

```diff
@@ -105,7 +105,8 @@ class RevertInfoCollection {
       match = m.group(1);
     }
     final String foundMatch = match!.trim();
-    final List<String> split = foundMatch.split(':');
-    return split.elementAt(1).trim();
+    // Explicit assignment for debugging.
+    final String matchedText = foundMatch.substring(foundMatch.indexOf(':') + 1).trim();
```

Comment:
Could we add a docstring somewhere to  indicate what section of text is being returned/should be returned. E.g. this is removing the "key" (prefixed by first ":") and returning the rest.

Reply:
Yeah I can do that. Good idea.